Earthfile

```diff
@@ -886,7 +886,12 @@ license:
 
 # npm-update-all helps keep all node package-lock.json files up to date.
 npm-update-all:
-    FROM node:16.16.0-alpine3.15
+    FROM node:24.9.0-alpine3.22
```

Comment:
could define a base `+node` target somewhere so we would only have to change one place the next time?

though with renovate perhaps that's not worth the complexity